controllers/kustomization_controller.go

```diff
@@ -142,6 +148,7 @@ func (r *KustomizationReconciler) Reconcile(req ctrl.Request) (ctrl.Result, erro
 			log.Error(err, "unable to update status")
```

Comment:
Github won't let me comment on L136 (after the ready condition is set to false due to progressing reason) but shouldn't we record readiness there since its value has potentially changed?

Reply:
Good catch! Fixed, thank you.